docs/src/site/resources_for_learners.md

```diff
@@ -12,3 +12,7 @@ section: "resources_for_learners"
 
 # Tutorials
  * [herding cats](http://eed3si9n.com/herding-cats/) by Eugene Yokota
+ * [Scala Exercises](https://www.scala-exercises.org/cats) by 47 Degrees 
+   offers multiple online exercises to learn about the typeclasses and 
```

Comment:
@raulraja Could you please change "typeclasses" to "type classes". We agreed on this spelling in #441. :-)